styles/style.css

```diff
@@ -81,20 +89,146 @@ textarea {
     appearance: none;
     border-radius: 0;
 }
+
 .btn {
     display: flex;
     justify-content: center;
-	align-items: center;
+    align-items: center;
     padding: 8px;
     width: 100%;
     max-width: 94px;
     border-radius: 2px;
     color: var(--color-white);
     font: var(--font-button-default);
 }
+
 .btn.btn--primary {
     padding: 16px;
     max-width: 163px;
     background: var(--color-accent);
     font: var(--font-button-small);
 }
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1177px;
+}
+
+.site-header .header-top {
+    padding: 14px 0;
+    background-color: #7E33E0;
+}
+.site-header .header-top a {
+    color: var(--color-white);
+    line-height: 18.75px;
+    font-size: 16px;
+    font-family: Arial, "Josefin Sans", sans-serif;
+}
+.site-header .header-top .container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 48px;
+
+}
+.site-header .header-top .container .header-left {
+    display: flex;
+    align-items: center;
+    gap: 48px;
+}
+.site-header .header-top .container .connection {
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    max-height: 16px;
+
+}
+
+.site-header .header-top .container .header-right {
+    display: flex;
+    align-items: center;
+    gap: 32px;
+}
+.site-header .header-top .container .header-right a {
+    display: flex;
+    align-items: center;
+    gap: 2px;
+}
+.site-header .header-top .container .header-right button {
+    border: unset;
+    background: transparent;
+}
+.header-bottom {
+    padding: 19px 0;
+}
+
+.header-bottom .container .header-bottom-row {
+    display: flex;
+    align-items: center;
+    background-color: var(--color-white);
+}
+.header-bottom .container .header-bottom-row .header-logo {
+  padding-right: 88px;
+}
+.header-bottom .container .header-bottom-row .header-nav {
+    display: flex;
+    justify-content: space-between;
```

Comment:
This styles doesnt worked. If you delete them nothing changes

Reply:
corrected